src/cargo/ops/cargo_package.rs

```diff
@@ -80,6 +80,8 @@ struct VcsInfo {
 #[derive(Serialize)]
 struct GitVcsInfo {
     sha1: String,
+    /// Indicate whether or not the Git worktree is dirty.
+    dirty: bool,
```

Comment:
Should we skip-on-serialize-if-default?  That way the file stays the same in the regular cases?

Reply:
That makes sense to me too. Just let me know when and if to make this change.

Reply:
I am fine with either.

Reply:
@torhovland feel free to go ahead and do it :)

Reply:
Updated.